src/database/models/instances.ts

```
@@ -0,0 +1,77 @@
+        instanceKeyHash,
+        userId,
+        description,
+        arweave_wallet_address: '',
+    }).returning();
+    return instance;
+}
+
+export async function updateInstanceKeyHash(
+    userId: string,
+    instanceKeyHash: string,
+): Promise<InstanceType | any> {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)